purchase_request/i18n/purchase_request.pot

```diff
@@ -0,0 +1,484 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
```

Comment:
As @pedrobaeza said and as it is currently beeing voted into the contributor mailing list, the .pot are no more necessay so you can remove it without any problems